pkg/utils/utils.go

```diff
@@ -12,6 +12,22 @@ import (
 	consolev1 "github.com/openshift/api/console/v1"
 )
 
+const (
+	baseURI       = "base-uri"
+	defaultSrc    = "default-src"
+	imgSrc        = "img-src"
+	fontSrc       = "font-src"
+	scriptSrc     = "script-src"
+	styleSrc      = "style-src"
+	consoleDot    = "console.redhat.com"
+	httpLocalHost = "http://localhost:8080"
+	wsLocalHost   = "ws://localhost:8080"
```

Comment:
Shouldn't we be building these from the passed-in args for off-cluster hostname and port when starting bridge?

Reply:
Could we open another bug/story for that and let this merge to cover the vast majority of use cases? I feel usage of off-cluster host customization is probably extremely rare and even in those cases, this will not introduce any regression. They will just continue to see CSP violations they are already seeing.